cmd/enterprise-portal/internal/database/internal/tables/tables.go

```diff
@@ -0,0 +1,14 @@
+package tables
+
+import (
+	"gorm.io/gorm/schema"
+
+	"github.com/sourcegraph/sourcegraph/cmd/enterprise-portal/internal/database/subscriptions"
+)
+
+// ⚠️ WARNING: This list is meant to be read-only.
+func AllTables() []schema.Tabler {
```

Comment:
```suggestion
func All() []schema.Tabler {
```

so just `tables.All()`?

Reply:
Fixed!